dask_kubernetes/core.py

```diff
@@ -537,6 +537,10 @@ def __exit__(self, type, value, traceback):
     def scheduler_comm(self):
         return self.cluster.scheduler_comm
 
+    @property
+    def worker_spec(self):
+        return self.cluster.worker_spec
```

Comment:
This shouldn't be necessary.  The distributed package shouldn't require this attribute.

Reply:
We use it a couple times in case we're dealing with a SpecCluster, but I had hoped that we weren't strict in our dependence.  In particular, dask-yarn doesn't work like this, and there aren't plans to make it do so in the future.

Reply:
Thanks for pointing that out. I've removed it